src/SLCore.UnitTests/Listener/Analysis/RaiseIssuesParamsTests.cs

```diff
@@ -83,15 +79,6 @@ [new ImpactDto(SoftwareQuality.MAINTAINABILITY, ImpactSeverity.LOW)])))
                          }
                        ]
                      },
-                    "severity": "BLOCKER",
```

Comment:
This is interesting. We SHOULD ignore these fields, however they've only been deprecated on the SLCore side, so they're still being sent. Should we have a test where we explicitly assert that we ignore these values?:

Reply:
Good point! Yes, sure. I will add one.